lib/BackendClient.js

```diff
@@ -467,6 +523,36 @@ var BackendClient = (function(){
         return deferred.promise;
     }
 
+    function updateRecordingStatus(entryId, entryServerNodeId, recordedEntryId, recordingStatus) {
+        let actionPromise = Q.defer();
+        logger.debug(`[${entryId}] Calling entryServerNode.update`);
+
+        let entryServerNode = new kalturaVO.KalturaLiveEntryServerNode();
+        let recordingInfo = new kalturaVO.KalturaLiveEntryServerNodeRecordingInfo();
+        recordingInfo.recordedEntryId = recordedEntryId,
+        recordingInfo.recordingStatus = recordingStatus;
+        let recordingInfoArray = [];
+        recordingInfoArray.push(recordingInfo);
+        entryServerNode.recordingInfo = recordingInfoArray;
+        
+        client.entryServerNode.update((result, err, headers) => {
```

Comment:
do we need impersonation here?

Reply:
No. If you think it's wrong need request Noam to change permission.